docs/manual/creating-a-checker.tex

```diff
@@ -357,6 +357,9 @@
   Write test cases for a few of the annotated JDK methods to ensure
   that the annotations are being properly read by your checker.
 
+  If there are errors in the Checker Framework's ``all-systems'' tests, you
+  may write your new qualifiers in the relevant test cases.
```

Comment:
The `all-systems` tests are not supposed to have any qualifiers.  See `framework/tests/all-systems/README`.